var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -174,7 +176,7 @@ class Ascent(CMakePackage, CudaPackage):
     #######################
     # VTK-m
     #######################
-
+    depends_on("vtk-m@2.0:", when="@0.9.2: +vtkh")
```

Comment:
@cyrush it looks like this makes no sense combined with the line below.

vtk-m 2.0 can never be picked cause any `0.9.2:` versions also matches `0.9.0:` which restricts it to vtk-m 1.9